### PR TITLE
Update lepton to 1.7.3

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by suschizu** [comment]
the developer didn't continue his apple developer account and the app hasn't a valid signature anymore … is there anything we can do here? 
https://github.com/hackjutsu/Lepton/releases

**Comment by vitorgalvao** [comment]
> is there anything we can do here?

No. But the problem here is that the new zip files appears malformed.

**Comment by core-code** [comment]
lets just change to the DMG file then

https://github.com/hackjutsu/Lepton/releases/download/v1.7.3/Lepton-1.7.3.dmg

**Comment by suschizu** [comment]
oh great - the dmg wasn't there when I updated the cask - great let's take this

File: Casks/lepton.rb [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
   # github.com/hackjutsu/Lepton was verified as official when first introduced to the cask
-  url "https://github.com/hackjutsu/Lepton/releases/download/v#{version}/Lepton-#{version}-mac.zip"
+  url "https://github.com/hackjutsu/Lepton/releases/download/v#{version}/Lepton-#{version}.dmg"
   appcast 'https://github.com/hackjutsu/Lepton/releases.atom'
   name 'Lepton'
   homepage 'https://hackjutsu.com/Lepton/'
 
   app 'Lepton.app'
-
-  zap trash: [
```
Comment (vitorgalvao) [inline Casks/lepton.rb]: Why remove the `zap`?

Comment (suschizu) [reply Casks/lepton.rb]: sorry - this was a mistake

**Pull request merged**